pages/index.js

```diff
@@ -43,6 +45,25 @@ const Home = () => {
               <div className="absolute-pos">
                 <h1>Ouro Payments</h1>
                 <h4>Privacy preserving continuous payment streams</h4>
+                <h4>
+                  Use Ouro for
+                  <TypistLoop interval={1000}>
+                    {[
+                      "Salary payments",
+                      "Car rental payments",
+                      "Service subscriptions",
+                    ].map((text) => (
+                      <Typist
+                        key={text}
+                        startDelay={500}
+                        cursor={{ show: false }}
```

Comment:
We can make the typing etc... behave differently, so let me know your thoughts on how it currently is and we can always easily tweak it